experimental/adaptive-tool/package.json

```diff
@@ -220,7 +220,9 @@
         "build:server": "webpack --mode production --config ./server/lg/webpack.config.js && webpack --mode production --config ./server/lu/webpack.config.js",
         "clean": "rimraf client/out && rimraf server/out",
         "postinstall": "cd client && npm install && cd ../server && npm install && cd ..",
-        "test": "sh ./scripts/e2e.sh"
+        "test": "sh ./scripts/e2e.sh",
+		"compile": "tsc -b",
```

Comment:
are these two lines necessary?

Reply:
just run build is enough, I have removed these two lines